CHANGELOG.md

```diff
@@ -0,0 +1,83 @@
+
+__Change__: Added a status callback for emails sent through SES.
+__Rational__: There was no way for us to get information if an email was delivered or not because SES does not send that information back automagically. Digging around the code base we found that there used to be an API callback from SES through SNS. This was removed and is now running in a lambda according to a GDS commit message with undisclosed source code. We added an API callback back in.
+__Commits__: https://github.com/cds-snc/notification-api/commit/3650bd86caa50fcfb482f9fc52bbd144b8715709
+
+__Change__: Changed the default templates in the database using migrations
+__Rational__: GDS hardcodes GOV.UK in their templates using database migrations. We had to add additional migrations to make our own changes to the template. We also made a migration use SNS as our preferred provider.
+__Commits__: https://github.com/cds-snc/notification-api/commit/447250f12f838b1f2e72c38b34299c281a19976f, https://github.com/cds-snc/notification-api/commit/f4b0f1a09486a47fc78e9d31d9952afc53c070c7, https://github.com/cds-snc/notification-api/commit/c622b5e4c8cce3b3e7aa0fe27b3ea55ca9adbf2f, https://github.com/cds-snc/notification-api/commit/c32a472d60e3f9ed1a328f4624685390b977893d
+
+__Change__: Change the CI/CD process for testing and building.
+__Rational__: CDS uses Google cloud build and Kubernetes engine. Added new docker files that use Alpine as their base vs. the debian images.
+__Commits__: https://github.com/cds-snc/notification-api/commit/5c7ffa4f215fd198cf7825cc733bf7dadb3bef7d
+
+__Change__: Repointed the requirements to use CDS utils vs. GDS utils
+__Rational__: We need to make changes to the utils repository and therefore forked it and repointed the API requirements to that.
+__Commits__: https://github.com/cds-snc/notification-api/commit/dfe9feb71015aff911f913bf717aad442576a2e4
+
+__Change__: Added Amazon SNS as a provider for sending text messages
+__Rational__: GDS currently only support Firetext and MMG to send text messages. CDS has no contract with those vendors but does have access to Amazon SNS as a text message vendor.
+__Commits__: https://github.com/cds-snc/notification-api/commit/c3305d55f7f8007841c045b77feb1f79e703932b, https://github.com/cds-snc/notification-api/commit/875f5d66fee2ce94af0a2a0bf167cfa9bc059c94, https://github.com/cds-snc/notification-api/commit/19e762e34320806c0a128747142e6d2196153c67
+
+__Change__: Refactored how environment variables get loaded and defaults are assigned.
+__Rational__: Environmental variables should be injected by the host (Docker, K8s pod, etc) into an application vs. hardcoded in a file. Sane defaults should exist if they are missing.
+__Commits__: https://github.com/cds-snc/notification-api/commit/0b6954c253151d3dbac1f62152c4de56e843d67c
+
+
+### Changes to ADMIN
+<https://github.com/cds-snc/notification-admin>
+
+__Change__:
```

Comment:
We should add removed 

https://github.com/alphagov/govuk_frontend_toolkit JavaScript and CSS/SASS dependancy removed - pull into the admin repo.

Reply:
Added